OpenEdXMobile/res/drawable/edx_enroll_button.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="false">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/edx_brand_gray_base" />
```

Comment:
@lizcohen 
Button is not looking disabled here with **edx_brand_gray_base** color  i-e- `#414141`
Here's the screen shot: 
<img src="https://cloud.githubusercontent.com/assets/3858265/26821901/64dce20a-4aa8-11e7-96a3-b4f54f1b5733.png" width="200">

thoughts ??